client/elife-theme/src/index.js

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable import/extensions */
 import './fonts/index.css'
 
+const gridUnit = 6
```

Comment:
Last time I checked PubSweet's gridUnit was the 'average', not the smallest possible spacing. So if we do this our components wont be compatible.

Reply:
Yes, but this is the new reality.

Reply:
Have you seen the latest eLife theme/styleguide in invision yet? https://projects.invisionapp.com/share/NZKXOQUV4P3#/screens/302926984